k8s/sandbox/cluster-00/jenkins-test/jenkins.yaml

```diff
@@ -319,16 +319,92 @@ spec:
                         shutdownOnIdle: false
                         storageAccountNameReferenceType: "existing"
                         storageAccountType: "Standard_LRS"
-                        subnetName: "jenkins-subnet"
+                        subnetName: "jenkins"
                         templateDesc: "Jenkins build agents for HMCTS"
                         templateDisabled: false
                         templateName: "cnp-jenkins-builders"
                         usageMode: "Use this node as much as possible"
                         usePrivateIP: true
                         virtualMachineSize: "Standard_D4_v3"
-                        virtualNetworkName: "mgmt-infra-sandbox"
-                        virtualNetworkResourceGroupName: "mgmt-infra-sandbox"
-
+                        virtualNetworkName: "core-sbox-vnet"
+                        virtualNetworkResourceGroupName: "core-infra-sbox-rg"
+                - azureVM:
+                    azureCredentialsId: "jenkinsServicePrincipal"
+                    cloudName: "cnp-azure-team-a"
+                    deploymentTimeout: 1200
+                    existingResourceGroupName: "mgmt-vmimg-store-sandbox"
```

Comment:
this might not work (we think it's where the VM agents will be deployed).  Maybe use a different one?

Reply:
for now, I've created this `mgmt-vmimg-store-sandbox-rpe999` for test purpose